mongo/integration/client_test.go

```diff
@@ -574,31 +572,35 @@ func TestClient(t *testing.T) {
 
 		// Assert that the minimum RTT is eventually >250ms.
 		topo := getTopologyFromClient(mt.Client)
-		assert.Soon(mt, func(ctx context.Context) {
+		callback := func() bool {
```

Comment:
Please apply the resolutions discussed in `TestClient/RTT90_is_monitored` to the remaining client tests.